tests/acceptance/server_span_test.py

```diff
@@ -29,7 +29,10 @@ def validate_span(span_obj):
 
     thrift_obj.side_effect = validate_span
 
-    TestApp(main({}, **settings)).get('/sample', status=200)
+    with mock.patch('pyramid_zipkin.request_helper.generate_span_id') \
```

Comment:
get_span() fixture returns id=1, so generate_span_id needs to return the same id for the test to pass.